pandas/util/_decorators.py

```diff
@@ -19,46 +19,42 @@ def deprecate(name, alternative, version, alt_name=None,
 
     Parameters
     ----------
-    name : str
-        Name of function to deprecate
-    alternative : str
-        Name of function to use instead
+    name : func
```

Comment:
Unless I am overlooking something I didn't see where this decorator was being used, but it seems like there is a mismatch between what was documented and the types it actually accepts (at least looking at `alternative`).

To make this work with substitution I took the liberty of assuming these arguments should functions and not strings whenever this decorator gets used

Reply:
> Unless I am overlooking something I didn't see where this decorator was being used, 

It's not used a lot, but at least it is used for argmin/argmax and for scatter_matrix (it is not used there as a decorator, so I searched for "deprecate("